parquet-variant/src/builder.rs

```diff
@@ -999,8 +1000,17 @@ impl<'a> ListBuilder<'a> {
         let offset_size = int_size(data_size);
 
         // Get parent's buffer
+        let offset_shift = match &self.parent_state {
```

Comment:
It might be nice if this was a function in ParentState, something like 

```suggestion
        let offset_shift = self.parent_state.object_start_offset();
```

Reply:
Fixed